content/src/controller/Controller.ts

```diff
@@ -669,12 +691,21 @@ async function setContentFileHeaders(content: RawContent, hashId: string, res: e
   res.setHeader('ETag', JSON.stringify(hashId)) // by spec, the ETag must be a double-quoted string
   res.setHeader('Access-Control-Expose-Headers', 'ETag')
   res.setHeader('Cache-Control', 'public,max-age=31536000,s-maxage=31536000,immutable')
+  res.setHeader('Accept-Ranges', 'bytes')
 
   if (content.encoding) {
     res.setHeader('Content-Encoding', content.encoding)
   }
 
-  if (content.size) {
+  if (content.size && content.range?.start !== undefined && content.range?.end !== undefined) {
```

Comment:
is it possible to `start` or `end` be undefined but still a valid range?

Reply:
ok, it is not possible but maybe is a good idea to have the optional range but required `start`/`end` when you create the `ContentItem`

Reply:
Thanks for pointing out this. I was able to come up with a more elegant solution here: https://github.com/decentraland/catalyst/pull/915/commits/67397820ff56ce06a1a7db63887c6a532a9d703f

The point was that the `range` that is used to create the headers is one that is calculated when reading the file. So, we have both start and end properties or none of them.